README.md

```diff
@@ -48,3 +48,15 @@ development environment you can use.
    ```bash
    docker-compose run --rm obs-docu daps -vv -d DC-obs-all html
    ```
+
+## Update documentation
+
+The main steps to update the documentation are:
```

Comment:
[nitpick]

Are there secondary steps to update the documentation? If not, then the word _main_ is unneeded here. If there are, then we should document them too.

Reply:
The first steps are here and the rest in `obs-landing`. I'll change it to do it clearer.